ui/apps/dashboard/src/app/(dashboard)/env/[environmentSlug]/functions/[slug]/replay/page.tsx

```diff
@@ -106,7 +106,9 @@ export default function FunctionReplayPage({ params }: FunctionReplayPageProps)
       environmentID: environment?.id!,
       functionSlug: functionSlug,
     },
+    context: useMemo(() => ({ additionalTypenames: ['Replay'] }), []),
```

Comment:
This is needed so that the replay list refreshes after a mutation when the list is empty.

See https://formidable.com/open-source/urql/docs/basics/document-caching/#document-cache-gotchas